site/src/components/card/Card.tsx

```diff
@@ -7,27 +7,36 @@ import {
 import clsx from "clsx";
 import Link from "@docusaurus/Link";
 import { useTheme } from "@jpmorganchase/uitk-core";
+import { TearOutIcon } from "@jpmorganchase/uitk-icons";
 
 import useOnScreen from "../../utils/useOnScreen";
 
 import styles from "./Card.module.css";
+import { isExternal } from "util/types";
+
+export interface FooterProps {
+  isExternalLink?: boolean;
+  footerText: string;
+}
 
 export interface CardProps extends ComponentPropsWithoutRef<"div"> {
-  icon: JSX.Element;
+  icon?: JSX.Element;
+  inlineIcon?: JSX.Element;
   title: string;
   description: string;
   url: string;
-  footer: string;
+  footer: FooterProps;
```

Comment:
I think in the future it would be good for this to accept either an object or a string but not a blocker :)